### PR TITLE
Improve the Release Manager's Guide

### PR DESCRIPTION
**Pull request by tomhukins**

### CODE REVIEW

**Review by khwilliamson — COMMENTED** [review]
Actually there is a dromedary server that replaces the outdated reference.  So we should direct people to that. @toddr can you give the correct reference

File: Porting/release_managers_guide.pod [hunk 1]
```diff
@@ -166,10 +160,6 @@ pre-release testing, and you may wish to upload to PAUSE via URL.
 Make sure you have a way of sharing files, such as a web server or
 file-sharing service.
 
-Porters have access to the "dromedary" server (users.perl5.git.perl.org),
```
Comment (toddr) [inline Porting/release_managers_guide.pod:169]: Honestly I'd recommend we just suggest a direct PAUSE upload here? do we really need an alternate place to upload the upload?

Comment (Grinnz) [reply Porting/release_managers_guide.pod:169]: I believe the intention of this space is a pre-PAUSE sanity check upload as often used by releasers.

Comment (tomhukins) [reply Porting/release_managers_guide.pod:169]: Thanks to @toddr I now have access to the new dromedary.  However, it looks like it has no per-user public_html support: this is disabled in `/etc/httpd/conf.d/userdir.conf`.  Although my commit message was wrong - we do have a dromedary server - I suggest keeping the contents of my commit to remove any mention of dromedary, but amending the commit message.  Does that seem sensible?

**Comment by tomhukins** [comment]
I'll merge fe5842c to blead in the next few days unless anyone objects.

**Pull request closed without merging**